src/app/src/lib/getaccs.ts

```diff
@@ -5,8 +5,9 @@ const { abi } = _abi
 
 
 function replaceId(str: string, id: number) {
-  const idHex = id.toString(16).padStart(64, '0');
-  return str.replace(/\{id\}/g, idHex);
+  // const idHex = id.toString(16).padStart(64, '0');
+  const paddedId = id.toString().padStart(64, '0')
```

Comment:
is there a reason you replaced `toString(16)` with `toString()`? I think the id needs to be in hexadecimal: https://eips.ethereum.org/EIPS/eip-1155

Reply:
yeah you're right. The initial implementation of the metadata during the hackathon was in decimal which is why I changed it but looking at the docs, I'll change the metadata urls in the backend